browser/js/pipeline/pipeline.stage/pipeline.stage.candidate/panelTemplates/kit.html

```diff
@@ -1,3 +1,18 @@
-<div class="kit-panel">
-{{data}}
-</div>
\ No newline at end of file
+<div class="kit-panel" ng-controller="kitPanelCtrl" >
+    <div class="question-field" ng-repeat="question in data.questions">
+    	<!-- @Omri passing in 
+    	data to angular directive within ng-repeat evaluates to undefined
+    	inside directive's isolate scope if attribute name is camel-case 
+    	i.e. data-question is undefined but changing to data is fine 
+		http://stackoverflow.com/questions/15754889/angular-directive-does-not-evaluate-inside-ng-repeat 
+		any thoughts why? -->
+        <kit-dropdown-field ng-model="model[question.id]" question="question" ng-if="question.selectedField == 'dropdown'"></kit-dropdown-field>
```

Comment:
Maybe consider making a `kit-widget` or `kit-input` directive? Just seems like a nice way to encapsulate "any possible kit input". It might not save you any lines of code but it just seems nice and modular.